test/unit_Polygon.jl

```diff
@@ -198,19 +198,19 @@ for N in [Float64, Float32, Rational{Int}]
             @test v1 ∈ h1
         elseif i == 2
             c = h1.constraints[1]
-            @test c.a ≈ to_N(N, [0.4, 0.5])&& c.b ≈ to_N(N, (0.46))
+            @test c.a ≈ to_N(N, [0.4, 0.5]) && c.b ≈ to_N(N, (0.46))
```

Comment:
is there sth special about this test that `to_N` is needed ? instead of `N[0.4, 0.5]`

Reply:
since

```julia
julia> N = Rational{Int}
Rational{Int64}

julia> N[0.4, 0.5]
2-element Array{Rational{Int64},1}:
 3602879701896397//9007199254740992
                1//2
```

Reply:
The numbers are very big. Sometimes this leads to overflow errors.
In this case there is no problem, but if we do computations with those numbers, I think it is better to make them small again to avoid problems if we eentually extend the tests.

Reply:
The automatic "good" conversion works for integers (and apparently for `0.5` as well, but I did not consider this case), so I removed the `to_N` for integers.

Reply:
OK, in those cases you prefer to use `rationalize`

```julia
julia> rationalize(0.4)
2//5

julia> Rational{Int}(0.4)
3602879701896397//9007199254740992
```

Reply:
as an alternative `N(4//10)` is `2//5` if `N = Rational{Int}` or `0.4` if `N` is a float.

Reply:
> OK, in those cases you prefer to use rationalize

This is what `to_N` does :+1: